Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -190,6 +190,8 @@ public async Task ValidateContestPassword(int id, bool official, string password
             return participationModel;
         }
 
+        public Task<IEnumerable<ContestForHomeIndexServiceModel>> GetAllPracticable() => throw new System.NotImplementedException();
```

Comment:
Why throw new System.NotImplementedException(); If the method is not used anymore, just delete it.

Reply:
It was forgotten... I successfully removed it and replaced it with the proper method name that was used.